jobbergate-api/jobbergate_api/apps/models.py

```diff
@@ -58,10 +60,22 @@ class IdMixin:
 
     Attributes:
         id: The id of the job script template.
+        cloned_from: Specify the id of the row that this row was cloned from.
     """
 
     id: Mapped[int] = mapped_column(Integer, primary_key=True)
 
+    @declared_attr
+    def cloned_from_id(cls) -> Mapped[int | None]:
```

Comment:
This needs to be added to the detailed and maybe list views on the response schemas, right now there is no way to visualize this value.

Reply:
solved in [885ab05](https://github.com/omnivector-solutions/jobbergate/pull/460/commits/885ab0582416b667b3f2f69e48febeb59547e47c)